src/pages/Organize/Uploads/Edit/index.js

```diff
@@ -14,22 +14,139 @@
  You should have received a copy of the GNU General Public License along
  with this program; if not, write to the Free Software Foundation, Inc.,
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-***************************************************************/
+*/
 
-import React from "react";
+import React, { useState, useEffect } from "react";
+import InputContainer from "../../../../components/Widgets/Input";
+import Button from "../../../../components/Widgets/Button";
+import { getFolders } from "../../../../services/getFolder";
+import { getUploadsFolderId } from "../../../../services/organizeUploads";
+import { getId } from "../../../../services/upload";
 
 const UploadEdit = () => {
+  const initialState = {
+    folderId: 1,
+    uploadId: null,
+    groupName: "",
+    uploadName: "",
+    uploadDescription: "",
+  };
+  const initialFolderList = [
+    {
+      id: 1,
+      name: "Software Repository",
+      description: "Top Folder",
+      parent: null,
+    },
+  ];
+  // const initialMessage = {
+  //   type: "success",
+  //   text: "",
+  // };
+  const [editUploadFolderData, setEditUploadFolderData] =
+    useState(initialState);
+  const [folderList, setFolderList] = useState(initialFolderList);
+  const [uploadFolderList, setUploadFolderList] = useState([]);
+  // const [loading, setLoading] = useState(false);
+  // const [showMessage, setShowMessage] = useState(false);
+  // const [message, setMessage] = useState(initialMessage);
+
+  const handleChange = (e) => {
+    setEditUploadFolderData({
+      ...editUploadFolderData,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  useEffect(() => {
+    getFolders().then((res) => {
+      setFolderList(res);
+    });
+    getUploadsFolderId(editUploadFolderData.folderId).then((res) => {
+      setUploadFolderList(res);
+    });
+    getId(editUploadFolderData.uploadId).then((res) =>
+      setEditUploadFolderData({
+        ...editUploadFolderData,
+        uploadName: res.uploadname,
+        uploadDescription: res.description,
+      })
+    );
+  }, [editUploadFolderData.folderId, editUploadFolderData.uploadId]);
+
   return (
-    <div className="main-container my-3">
-      <div className="row">
-        <div className="col-lg-8 col-md-12 col-sm-12 col-12">
-          <h1 className="font-size-main-heading">
-            Edit Uploaded File Properties
-          </h1>
-          <br />
+    <>
```

Comment:
Is this expected here?

Reply:
actually this is React fragment I checked the code its fine